pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -627,8 +627,8 @@ func (c *GettableUserConfig) GetGrafanaReceiverMap() map[string]*GettableGrafana
 }
 
 type GettableApiAlertingConfig struct {
-	Config `yaml:",inline"`
-
+	Config              `yaml:",inline"`
+	MuteTimeProvenances map[string]models.Provenance `yaml:"muteTimeProvenances,omitempty" json:"muteTimeProvenances,omitempty"`
```

Comment:
This may not be too important since templates are stored at a different level in the JSON. This puts the provenances right next to the existing `mute_timings` field right? If so, this is probably fine.

Reply:
Yes exactly, I saw that for templates it was one layer above, but I wanted to have it by the mute timings, otherwise, it's strange when using it and looking it up.